README.md

```diff
@@ -102,6 +107,21 @@ What is right matters, why go to justice (rather than...), what is fair is disce
 Extra Reading:
 - [Justice (SEP)](https://plato.stanford.edu/entries/justice/)
 
+## Facilitation
+
+Here are some guidelines about preparation in order to facilitate a session:
+
+- Before the session: 
+  - Have read and thought about the texts
+  - Identify and write 2-4 "themes" or questions you are interested in from the readings
+  - We use the same [shared notepad](https://hackmd.io/oEcuKALCTi-PbawLmT_Ixw#): https://hackmd.io/oEcuKALCTi-PbawLmT_Ixw#, which has a template for notes
+- During the session:
+  - Arrive on time (call should auto record as the first person enters)
```

Comment:
Given today’s experience, maybe recording be another bullet:

> - Ensure the call is recorded (it should auto-record as the first person enters, but always make sure someone presses the record button if not)

Reply:
thanks for the review and suggestion, I've incorporated!